cfn-resources/alert-configuration/cmd/resource/resource.go

```diff
@@ -316,27 +326,24 @@ func expandAlertConfigurationNotification(notificationList []NotificationView) (
 	}
 
 	for ind := range notificationList {
-		notification := mongodbatlas.Notification{
-			APIToken:            cast.ToString(notificationList[ind].ApiToken),
-			ChannelName:         cast.ToString(notificationList[ind].ChannelName),
-			DatadogAPIKey:       cast.ToString(notificationList[ind].DatadogApiKey),
-			DatadogRegion:       cast.ToString(notificationList[ind].DatadogRegion),
-			EmailAddress:        cast.ToString(notificationList[ind].EmailAddress),
+		notification := atlasSDK.AlertsNotificationRootForGroup{
+			ApiToken:            notificationList[ind].ApiToken,
+			ChannelName:         notificationList[ind].ChannelName,
+			DatadogApiKey:       notificationList[ind].DatadogApiKey,
+			DatadogRegion:       notificationList[ind].DatadogRegion,
+			EmailAddress:        notificationList[ind].EmailAddress,
 			EmailEnabled:        notificationList[ind].EmailEnabled,
-			FlowdockAPIToken:    cast.ToString(notificationList[ind].FlowdockApiToken),
-			FlowName:            cast.ToString(notificationList[ind].FlowName),
```

Comment:
@Zuhairahmed I could not locate this alert configuration in the API doc and these are not supported by the new SDK. Do you have more insights into this?
 
https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Alert-Configurations/operation/createAlertConfiguration

Reply:
Hi @maastha, these notification options were deprecated a while back and are no longer available in atlas. Feel free to remove them

Reply:
removed